backend/app/notify/telegram.go

```diff
@@ -188,6 +190,43 @@ func escapeTelegramText(text string) string {
 	return text
 }
 
+// telegram not allow h1-h6 tags
+// replace these tags with a combination of <b> and <i> for visual distinction
+// https://github.com/umputun/remark42/issues/1202
```

Comment:
Just a little note, I think that particular link shouldn't be there: we don't have them elsewhere in the code, and it would be easy to deduct from the git annotation. If understanding a function requires you to go through the link, we are likely doing something wrong, and here nothing will break if we omit this link.

Reply:
If you're wondering, we use links in code for external API reference and source attribution when the code is copied from somewhere.